dietpi/dietpi-software

```diff
@@ -14205,6 +14231,18 @@ If no WireGuard (auto)start is included, but you require it, please do the follo
 
 		fi
 
+                software_id=203 # Zerotier
+                if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == -1 )); then
+                        Banner_Uninstalling
+			#Purging down all information 
+                        [[ -f '/usr/sbin/zerotier-cli' ]] && G_EXEC rm /usr/sbin/zerotier-cli
+                        [[ -f '/usr/sbin/zerotier-one' ]] && G_EXEC rm /usr/sbin/zerotier-one
+                        [[ -f '/etc/apt/sources.list.d/zerotier.list' ]] && G_EXEC rm /etc/apt/sources.list.d/zerotier.list
+                        [[ -f '/var/lib/zerotier-one/identity.public' ]] && G_EXEC rm /var/lib/zerotier-one/identity.public
+                        [[ -f '/etc/init.d/zerotier-one' ]] && G_EXEC rm /etc/init.d/zerotier-one
+			G_EXEC apt remove zerotier-one -y
```

Comment:
you would need to purge the package first before removing files. As well, pls use our own function like `G_AGP` instead of `apt remove`

Reply:
I didn't understand all the shortcuts if there are file with command and descriptions could you please show me?